bd-runtime/src/runtime.rs

```diff
@@ -747,6 +747,28 @@ pub mod resource_utilization {
   );
 }
 
+pub mod session_replay {
+  use time::ext::NumericalDuration as _;
+
+  bool_feature_flag!(
+    PeriodicWireframesEnabledFlag,
+    "session_replay.periodic_wireframes.enabled",
+    false
+  );
+
+  bool_feature_flag!(
+    ScreenshotsEnabledFlag,
+    "session_replay.screenshots.enabled",
+    false
+  );
+
+  duration_feature_flag!(
+    ReportingIntervalFlag,
+    "session_replay.periodic_wireframes.reporting_interval_ms",
+    3.seconds()
```

Comment:
ms or seconds?

Reply:
also currently this is passed by the user during start(), are you proposing to remove that capability altogether and only be able to control it via runtime var?